javascript/javascript1/week4/voice-assistant.js

```diff
@@ -0,0 +1,82 @@
+const myName = [];
+const toDoList = [];
+function getReply(command) {
+  // for (let i = 0; i < myName.length; i++)
+  if (command.includes("Hello my name is")) {
+    const newArr = command.split(" ");
+    myName.push(newArr[newArr.length - 1]);
+    console.log(myName);
+    return `Nice to meet you ${myName[myName.length - 1]}`;
+  }
+
+  // ===>>> This part of the code works, but doesn`t allow the function to execute the code further
```

Comment:
which part ? line 13-14? or rest of the code?

Reply:
Line 13-14 (together with line 4)